bagitutils.py

```diff
@@ -37,7 +37,7 @@ def __init__(self):
 # Column names in Boardwalk that cannot simply be copied over to FireCloud;
 # they require extra logic.
 COMPLEX_COLUMNS = FILE_COLUMNS + [
-    BoardwalkColumns.DONOR_UUID,
+    #BoardwalkColumns.DONOR_UUID,
```

Comment:
I would just delete rather than commenting out, but your choice. My feeling is you can always retrieve with Git history, and it just makes the code harder to read.